_posts/05-04-01-Design-Patterns.md

```diff
@@ -1,18 +1,18 @@
 ---
 isChild: true
-title: Dizajn paterni (šabloni)
+title: Dizajn šabloni (Design patterns)
 anchor: design_patterns
 ---
 
-## Dizajn paterni (šabloni) {#design_patterns_title}
+## Dizajn šabloni (Design patterns) {#design_patterns_title}
```

Comment:
Pa da, u pravu si ne znam zašto sam to ostavio tako, valjda da bi ljudi koji su totalni noob-ovi za engleski jezik, ipak znali šta znači to pattern, jer će im onda bar malo biti jasnija uloga, svrha toga. :) Ali naravno, obavezno sam ostavio Design patterns u zagradi, pošto su se sa tim susretali. 

Možda samo da obrnem to što je u zagradi i ovo van nje? Mada možda bolje da izbacim u poptunosti to "šablon", "šabloni"...